scripts/create_alliance.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+NEW_DENOM=$1
+KEY_NAME="me"
+BINARY_DIR=".noria"
+CHAIN_ID="oasis-3"
+DENOM="unoria"
+GAS_PRICE_DENOM="ucrd"
+GAS_PRICE="0.0025"
+NODE="http://127.0.0.1:26657/"
+export DAEMON_NAME="noriad"
+export DAEMON_HOME="$HOME/$BINARY_DIR"
+
+# Block height and sender key name must be set
+if [ -z "$1" ]; then
+  echo "Parameter new denom is missing"
+  exit 1
+fi
+
+exe() { echo "EXE\$ $@" ; "$@" ; }
```

Comment:
This is super neat! Nice way to describe what each step is doing during execution.